### PR TITLE
Add support for fetching secrets in ASP.NET JSON format

### PR DESCRIPTION
**Pull request by ryan-blunden**
Add support for fetching secrets in ASP.NET JSON format.

Closes ENG-5049

### CODE REVIEW

File: pkg/models/secrets_format.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```
Comment (Piccirello) [inline pkg/models/secrets_format.go]: Can you revert this?

File: pkg/models/secrets_format.go [hunk 2]
```diff
@@ -21,30 +21,32 @@ type SecretsFormat int
 // the source of the value
 const (
 	JSON SecretsFormat = iota
+	DOTNET_JSON
 	ENV
 	YAML
 	DOCKER
-	ENV_NO_FILE
+	ENV_NO_QUOTES
```
Comment (Piccirello) [inline pkg/models/secrets_format.go:28]: Can you move this rename into a separate commit? You'll want to prefix it with `chore: ` so it doesn't show in the changelog.

**Review by Piccirello — COMMENTED** [review]
Can you make the commit message slightly more descriptive? Something like `Add support for fetching secrets in dotnet format`

File: pkg/models/secrets_format.go (hunk 1 above)
Comment (ryan-blunden) [reply pkg/models/secrets_format.go]: Yep. VS Code did that on save.

**Review by Piccirello — DISMISSED** [review]

**Comment by ryan-blunden** [comment]
@Piccirello Any ideas for fixing the Salus issue? I've looked at the code in question, but a solution isn't readily apparent to me.

**Comment by Piccirello** [comment]
> @Piccirello Any ideas for fixing the Salus issue? I've looked at the code in question, but a solution isn't readily apparent to me.

I put up #334 for this. Once that's merged, you can rebase onto `master` and force push - that should fix it.

**Review by Piccirello — APPROVED** [review]
